src/cli/commands/lint.ts

```diff
@@ -69,9 +74,21 @@ linting ./openapi.yaml
       }
     }
 
+    let config: ILintConfig = mergeConfig(createEmptyConfig(), flags);
+
+    const configFile = configFileFlag || getDefaultConfigFile(process.cwd()) || null;
+    if (configFile) {
+      try {
+        const loadedConfig = await loadConfig(configFile);
+        config = mergeConfig(loadedConfig, flags);
+      } catch (ex) {
+        this.error('Cannot load provided config file');
```

Comment:
I think it would be helpful to actually log the `ex` in there. That could give the developer a hint of what's happening.
```suggestion
        this.error('Cannot load provided config file', ex);
```

Reply:
done